### DISCUSSION TITLE
Using Custom tile

### DISCUSSION THREAD

**Discussion by UmangSharma5** [openseadragon/openseadragon#2381] [Q&A]
I have ran the dzsave to convert my .tif file to .dzi and now i have deleted the _images 
directory and instead trying to use 

`  <Tile Level="13" Overlap="1" Column="2" Row="8" tileUrl="https://play.min.io:9000/umangsstudentsiiitacin/test/tmp/india_files/13/14_9.jpeg?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=Q3AM3UQ867SPQQA43P2F%2F20230712%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230712T205158Z&X-Amz-Expires=180000&X-Amz-SignedHeaders=host&X-Amz-Signature=b980ca68fff7c7d2e02fdc57b35da40723afd6e0ec5cf062c8454b6101a14264" />
`

Here i am trying to use tileUrl  , i have updated the .dzi file

and in my script.js i am running 

OpenSeadragon({
    id: "openseadragon1",
    prefixUrl: "./openseadragon/images/",
    tileSources: "india.dzi"
});

But the issue is because of the default behaviour of openseadragon , when i see my network tab it trirs to request locally http://127.0.0.1:5500/india_files/11/4_1.jpeg

How can i make sure it requests to tilUrl which i have provided instead of locally

**Comment by iangilman** [comment]
So you want to use a DZI but you have special URLs for the tiles themselves? You should be able to use some of the suggestions from here: 

https://github.com/openseadragon/openseadragon/discussions/2377

The DZI format that OSD supports does not include a `<Tile>` element.

**Comment by UmangSharma5** [comment]
So sir we can't use special url because all the india_files will be stored in minio , so whenever i want to use OSD to see my image i don't want to user to store the india_files in there system instead i want them to store it in cloud(minio) and create unique link for each of them and then try to change my .dzi . So is it possible to implement is idea using OSD

**Reply by iangilman** [reply]
Yes, either of the techniques described in #2377 should allow you to use the DZI off of one place and the tiles from another place.

**Comment by UmangSharma5** [comment]
Thank you so much it worked !!!!
